__test__/integration/token_api.spec.ts

```diff
@@ -160,6 +184,44 @@ describe("Token API (Integration Tests)", () => {
         });
 
         describe("contract exists at token address", () => {
+            describe("sender has insufficient balance", () => {
+                beforeEach(async () => {
+                    await dummyZRXToken.approve.sendTransactionAsync(OPERATOR, new BigNumber(10), {
+                        from: SPENDER,
+                    });
+
+                    await dummyZRXToken.setBalance.sendTransactionAsync(SPENDER, Units.ether(0), {
+                        from: CONTRACT_OWNER,
+                    });
+                });
+
+                test("should throw INSUFFICIENT_SENDER_BALANCE", async () => {
+                    await expect(
+                        tokenApi.transferFromAsync(
+                            dummyZRXToken.address,
+                            SPENDER,
+                            RECIPIENT,
+                            new BigNumber(10),
+                            { from: OPERATOR },
+                        ),
+                    ).rejects.toThrow(TokenAPIErrors.INSUFFICIENT_SENDER_BALANCE());
+                });
+            });
+
+            describe("sender has insufficient allowance", async () => {
```

Comment:
Please change description to `"spender has given operator insufficient allowance"`